Modules/Sources/Yosemite/Tools/Media/ExportableAsset.swift

```diff
@@ -1,4 +1,5 @@
 import Photos
+import UIKit
```

Comment:
It's surprising to see this one imported in Yosemite... but I see it's needed for UIImage. I guess we got this import for free before...

Reply:
Yes. It came from the umbrella header:

https://github.com/woocommerce/woocommerce-ios/blob/e94edaa925b582220dbdef01252486429f066e55/Yosemite/Yosemite/Yosemite.h#L1